layers/layer_options.cpp

```diff
@@ -1045,6 +1046,11 @@ void ProcessConfigAndEnvSettings(ConfigAndEnvSettings *settings_data) {
                                     gpuav_settings.shader_instrumentation.post_process_descriptor_index);
         }
 
+        if (vkuHasLayerSetting(layer_setting_set, VK_LAYER_GPUAV_VERTEX_ATTRIBUTE_FETCH_OOB)) {
+            vkuGetLayerSettingValue(layer_setting_set, VK_LAYER_GPUAV_VERTEX_ATTRIBUTE_FETCH_OOB,
+                                    gpuav_settings.shader_instrumentation.vertex_index);
```

Comment:
```suggestion
                                    gpuav_settings.shader_instrumentation.vertex_attribute_fetch_oob);
```

no reason to not align the names